CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 You can find and compare releases at the [GitHub release page](https://github.com/nuwave/lighthouse/releases).
 
 ## Unreleased
+- Replaced `Carbon\Carbon` with `Illuminate\Support\Carbon`
```

Comment:
That belongs in the `### Changed` section and should be active voice and present tense.